features/step_definitions/logging.rb

```diff
@@ -993,16 +993,14 @@
   pod_label = "app=elasticsearch-server"
 
   # create secret/elasticsearch-server for ES if needed
-  if transport_ssl_enabled == "true" || scheme == "https"
+  if scheme == "https"
```

Comment:
I'd like use http_ssl_enabled instead of scheme

Reply:
Using `http_ssl_enabled` makes it hard to figure out the external ES url when creating the CLF/instance:
```
    When I process and create:
      | f | <file> |
      | p | URL=<scheme>://elasticsearch-server.<%= cb.es_proj.name %>.svc:9200 |
    Then the step should succeed
```

Reply:
I mean using ( transport_ssl_enabled == "true" ). I saw you had init transport_ssl_enabled  at L981. 
https://github.com/openshift/verification-tests/blob/edb0cc137ede922e4d0ff1e76e7cce55b9fa11bf/features/step_definitions/logging.rb#L981

Reply:
I am thinking the scheme can quit the scene once we init transport_ssl_enabled

Reply:
Oh, I didn't notice that, will update the step, thanks.